lib/gobstones/expectations/has_procedure_usage.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../with_tempfile'
+
+module Gobstones::Expectations
+  class HasProcedureUsage
+    include Gobstones::WithTempfile
```

Comment:
I don't... in the intermediate version the tempfile was created here, and I forgot to delete it.